api/app/auto_spatial_advisory/elevation.py

```diff
@@ -262,12 +263,17 @@ async def process_tpi_by_firezone(run_type: RunType, run_date: date, for_date: d
 
     fire_zone_stats: Dict[int, Dict[int, int]] = {}
     async with get_async_write_session_scope() as session:
-        stmt = text("SELECT id, source_identifier FROM advisory_shapes;")
+        stmt = text("SELECT id, source_identifier FROM public.advisory_shapes;")
         result = await session.execute(stmt)
 
         for row in result:
             output_path = f"/vsimem/firezone_{row[1]}.tif"
-            warp_options = gdal.WarpOptions(format="GTiff", cutlineDSName=DB_READ_STRING, cutlineSQL=f"SELECT geom FROM advisory_shapes WHERE id={row[0]}", cropToCutline=True)
+
+            input_srs = osr.SpatialReference()
```

Comment:
You might be able to simplify this to:
`input_srs = hfi_masked_tpi.GetSpatialRef()`

Reply:
Done in: https://github.com/bcgov/wps/pull/3995/commits/8ab763f4855d688383d29cd8c6646904478daf5e